en/04_Changelogs/4.12.0.md

```diff
@@ -20,6 +21,28 @@ While it is still advised that you perform your own due diligence when upgrading
 
 ## Features and enhancements {#features-and-enhancements}
 
+### Search multiple `searchable_fields` by default {#general-search-field}
+
+The general search text field which appears in the search interface now searches across all of your searchable fields by default. It will return a match if the search terms appear in any of the searchable fields.
```

Comment:
```suggestion
The general search text field which appears in the search interface now searches across all of your searchable fields by default. It will return a match if the search terms appears in any of the searchable fields.
```

Reply:
"search terms" is plural. "search terms appear" is grammatically correct.